doc/system_requirements.md

```diff
@@ -20,6 +20,10 @@ The EM main goal is to simplify installing, configuring and updating extensions
 
 Database Administrators (DBA) use EM through it's REST API or a user interface for managing extensions in their Exasol databases.
 
+### Extension Developers
```

Comment:
I love roles being considered and described in requirements.

Reply:
:+1: Helps to define privileges and responsibilities. Especially in systems that implement access control.